addons/spectrum/functions/fnc_addSoundSequenceServer.sqf

```diff
@@ -0,0 +1,116 @@
+#include "script_component.hpp"
+/*/////////////////////////////////////////////////
+Author: b-mayr-1984 - Bernhard Mayr
+		(leaning heavily on Crowdedlight's work in fnc_addRandomRadioTrackingChatterServer.sqf)
+			   
+File: fnc_addSoundSequenceServer.sqf
+
+Parameters:  
+	_unit		unit/object to attach the signal to
+	_freq		frequency to broadcast on
+	_range		range of the broadcasted signal
+	_sounds		array of sounds to be played/transmitted in sequence
+					alternatively, instead of a sound a number can be given (then no sound is played, just silence for that duration in seconds)
+	_loop		boolean that tells if the sequence shall loop indefinately (default: false); useful for permanently active radio broadcasting stations
+
+Return:  script handle of spawned loop (can be used to terminate the sound sequence by using "terminate _handle;")
+
+Use this function 
+to add a list of sounds (CfgSounds or a sound file; see BIKI on playSoundUI for details)
+to be broadcast in the spectrum.
+
+Server only
+
+Examples:
+	_audioSequenceLoop = [_myRadioTower, 98.5, 5000, ["News_Jingle", "News_idap"], true] call crowsew_spectrum_fnc_addsoundsequenceserver;
+
+	_audioSequenceLoop = [_myRadioTower, 98.5, 5000, ["News_Jingle", 1.5, "News_idap", 1.1], true] call crowsew_spectrum_fnc_addsoundsequenceserver;
+
+	// to stop the sequence later make these 2x calls:
+	terminate _audioSequenceLoop;
+	[_myRadioTower, "sound"] call crowsew_spectrum_fnc_removebeaconserver;
+
+*///////////////////////////////////////////////
+
+if (!isServer) exitWith {};
+
+params [["_unit", objNull, [objNull]], ["_freq", 123, [1337]], ["_range", 300, [1337]], ["_sounds", [], [[]]], ["_loop", false, [false]]];
+
+if (isNull _unit || _sounds isEqualTo []) exitWith {
+	diag_log format ["CrowsEW:fnc_addSoundSequenceServer.sqf: Can not add sequence with _unit=%1 or _sounds=%2.", _unit, _sounds]; 
+};
+
+// if the unit already plays a sound sequence stop it first
+private _existingHandle = _unit getVariable[QGVAR(radioSoundHandle), scriptNull];
+if (!isNull _existingHandle) then {
+	terminate _existingHandle;
+	// remove signal 
+	[_unit, "sound"] call FUNC(removeBeaconServer);
+
+	// remove from zeus draw list - We wait with publish update until end of script where we publish anyway for the new draws
+	// private _rmIndex = GVAR(radioTrackingAiUnits) findIf { (_x select 0) == _unit};
+	// GVAR(radioTrackingAiUnits) deleteAt _rmIndex;
```

Comment:
Is this commented out on purpose?

Reply:
Yes. I copied the content of `fnc_addRandomRadioTrackingChatterServer.sqf` as a starting point for this file. The commented out sections are from your code.

I left it in because I thought that I might add some Zeus 3D info later on (e.g. the sound name rendered in the red 3D text).
Maybe best to put a // TODO on top of those lines.

Reply:
All good! forgot I had those commented out in the other file. 😅 
yeah, a TODO might be prudent. I can add that when I merge.